client/flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -49,6 +50,8 @@
 		97C147021CF9000F007C117D /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		A4451C7667BDCD6E1412C05D /* Pods-Runner.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Runner.release.xcconfig"; path = "Target Support Files/Pods-Runner/Pods-Runner.release.xcconfig"; sourceTree = "<group>"; };
 		C22FBAFC61DB62AFF6C1342D /* Pods-Runner.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Runner.debug.xcconfig"; path = "Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"; sourceTree = "<group>"; };
+		C2308E142431A8A1003B2CD8 /* Runner.entitlements */ = {isa = PBXFileReference; lastKnownFileType = text.plist.entitlements; path = Runner.entitlements; sourceTree = "<group>"; };
+		C2308E152432A137003B2CD8 /* GoogleService-Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; name = "GoogleService-Info.plist"; path = "../../../../../../../../Library/Mobile Documents/com~apple~CloudDocs/GoldCat/Wingtip/WHO APP/Staging FB Config/GoogleService-Info.plist"; sourceTree = "<group>"; };
```

Comment:
Can't submit - breaks build

Reply:
unresolving this comment since this is still here... build fails on this

Reply:
Really don't understand how Xcode snuck this back in. Removing again and will ensure it's gone when pushed.

Reply:
Really don't understand how Xcode snuck this back in. Removing again and will ensure it's gone when pushed.

Reply:
Really don't understand how Xcode snuck this back in. Removing again and will ensure it's gone when pushed.